### PR TITLE
[KYUUBI #2029] Hive Backend Engine - Operation Logs

### PR DESCRIPTION
**Pull request by KenjiFujima**
### _Why are the changes needed?_

Hive Backend Engine - Operation Logs.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/operation/ExecuteStatement.scala [hunk 1]
```diff
@@ -31,6 +32,11 @@ class ExecuteStatement(
     override val shouldRunAsync: Boolean,
     queryTimeout: Long)
   extends HiveOperation(OperationType.EXECUTE_STATEMENT, session) {
+
+  private val operationLog: OperationLog = OperationLog.createOperationLog(session, getHandle)
```
Comment (yaooqinn) [inline externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/operation/ExecuteStatement.scala:36]: this is not necessary. this operation file will never be visit at read side.

I have already reimplemented the interface, we shall add a unit test for this to verify it's ok or not(which we shall find out why)

https://github.com/apache/incubator-kyuubi/blob/f22a14f1e50af6faad2a64902367af3f63a4ffc9/externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/operation/HiveOperationManager.scala#L96-L128

Comment (KenjiFujima) [reply externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/operation/ExecuteStatement.scala:36]: @yaooqinn, I'm confusing about whether the `Log4j12DivertAppender` needs the operation. The operation log is used for setting the current operation log.

**Comment by KenjiFujima** [comment]
@yaooqinn @pan3793, could you please help to review this again?

**Comment by yaooqinn** [comment]
This PR seems wrong,

1. The PR is about to fix/support hive operation log according to the PR title, but the implementation does not tough any hive engine side code.
2. the server log root use a different base dir to logs of engines, this PR seems to remove this part

**Comment by yaooqinn** [comment]
closed by https://github.com/apache/incubator-kyuubi/pull/2586

**Pull request closed without merging**